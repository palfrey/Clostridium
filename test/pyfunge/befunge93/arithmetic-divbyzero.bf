40#v/0" = 0/4">:#,_$.25*, v
   >025*"detcelfer orez yb noisivid">:#,_@
30#^%0" = 0%3">:#,_$.25*,@>
