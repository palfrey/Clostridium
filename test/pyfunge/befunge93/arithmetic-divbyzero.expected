What do you want 4 / 0 to be? 4/0 = 42 
What do you want 3 % 0 to be? 3%0 = 54 

